### DISCUSSION TITLE
May I prepare a new release 0.24.3?

### DISCUSSION THREAD

**Discussion by rubenporras** [eclipse/lsp4e#857] [General]
Hi @mickaelistria,

I would like to prepare a new LSP4E release, so that we can rollout to production the latest fixes that have been made in the last three weeks.

Would that be okay with you?

Regards

**Comment by mickaelistria** [comment]
I don't have any objection. Usually, I might object if there is complex ongoing work that is in a current too incomplete or fragile state to deliver safely, but that's not the case here (Wild Web Developer even already consumes snapshots)
